### DISCUSSION TITLE
ParaView Python issue with `pkg_resources`

### DISCUSSION THREAD

**Discussion by earlsioson** [Kitware/trame#128] [Q&A] [answered]
Hello @jourdain, how were you able to get around this issue?

[paraview issue](https://gitlab.kitware.com/paraview/paraview/-/issues/21393)

I'm getting that error on macOS 12.6 x86 using pyenv installed 3.9.5. Also tried 3.9.13. I get the error when running pvpython for a trame app.

**Comment by jourdain** [comment, marked as answer]
I edit the python file within my ParaView that I've downloaded and [empty the method like described in that issue](https://gitlab.kitware.com/paraview/paraview/-/issues/21393#note_1186305).

**Reply by jourdain** [reply]
BTW we are trying to fix it somehow in our future PV binaries but we are still unsure on how to go about it. Hopefully it will make it to 5.11.

**Reply by earlsioson** [reply]
Sounds good, thanks again @jourdain
